hivemq-edge/src/frontend/src/extensions/datahub/components/nodes/ClientFilterNode.tsx

```diff
@@ -18,25 +18,23 @@ export const ClientFilterNode: FC<NodeProps<ClientFilterData>> = (props) => {
       <NodeWrapper route={`node/${type}/${id}`} {...props}>
         <HStack>
           <VStack>
-            <Text> {t('workspace.nodes.type', { context: type })}</Text>
+            <Text data-testid={'node-title'}> {t('workspace.nodes.type', { context: type })}</Text>
```

Comment:
suggestion(non-blocking): small little change on the attribute definitiona

```suggestion
            <Text data-testid="node-title"> {t('workspace.nodes.type', { context: type })}</Text>
```

Reply:
Lots of them around. I just realised I didn't configure eslint/prettier for `react/jsx-curly-brace-presence`.
Will fix the config and run a cleaning operation in a separate ticket